--- src/client/pkg/questions/types_reg_gen.go.orig	2025-02-19 16:40:07
+++ src/client/pkg/questions/types_reg_gen.go	2025-02-19 16:40:23
@@ -3,18 +3,14 @@
 
 import (
 	"github.com/widmogrod/mkunion/x/shared"
-	"slices"
-	"testing"
 )
 
 func init() {
-	shared.TypeRegistryStore[Contains[[]string,string]]("Contains[[]string,string]")
 	shared.TypeRegistryStore[MultipleChoiceEngToLatQuestion]("MultipleChoiceEngToLatQuestion")
 	shared.TypeRegistryStore[MultipleChoiceLatToEngQuestion]("MultipleChoiceLatToEngQuestion")
 	shared.TypeRegistryStore[ParseWordCompToLatQuestion]("ParseWordCompToLatQuestion")
 	shared.TypeRegistryStore[ParseWordLatToCompQuestion]("ParseWordLatToCompQuestion")
 	shared.TypeRegistryStore[PrincipalPartsQuestion]("PrincipalPartsQuestion")
-	shared.TypeRegistryStore[T]("T")
 	shared.TypeRegistryStore[TypeInEngToLatQuestion]("TypeInEngToLatQuestion")
 	shared.TypeRegistryStore[TypeInLatToEngQuestion]("TypeInLatToEngQuestion")
 }
